### PR TITLE
feat: [DHIS2-9156] Add TEI reserved value to user sync test

### PR DESCRIPTION
**Pull request by Sharmyn28**
**Description**
Add the number of TEI reserved value to user sync test

jira issue: [DHIS2-9156](https://jira.dhis2.org/browse/DHIS2-9156)

**Solution description**   
- Get value from datastore (general_settings, teiReservedValue)
- Show value and recommended value

### CODE REVIEW

File: src/components/sections/user-sync-test/run-test.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ const RunTest = ({ states }) => {
                                         {test.title}
                                     </p>
                                     <p className={itemStyles.subItemItem}>
-                                        {i18n.t('Recommended maximum:')}{' '}
+                                        {i18n.t('Recommended maximum:')}
```
Comment (amcgee) [inline src/components/sections/user-sync-test/run-test.js:48]: This should use an i18next interpolation variable (`i18n.t('Recommended maximum: {{maxValue}}', { maxValue: test.maxValue })`)

**Review by amcgee — APPROVED** [review]
This logic is very complex and these functions should be simplified, but the change looks OK (made one minor suggestion)

File: src/components/sections/user-sync-test/run-test.js (hunk 1 above)
Comment (Sharmyn28) [reply src/components/sections/user-sync-test/run-test.js:48]: Hi Austin! When I use interpolation I get this:

image

**Comment by Sharmyn28** [comment]
> This logic is very complex and these functions should be simplified, but the change looks OK (made one minor suggestion)

Great! I'm working on refactoring the code but as a part of another PR :)

File: src/components/sections/user-sync-test/run-test.js (hunk 1 above)
Comment (amcgee) [reply src/components/sections/user-sync-test/run-test.js:48]: Sorry @Sharmyn28, my fault!  This is due to the `:` character being used as the namespace separator.  You can avoid this with (I believe):

```js
i18n.t('Recommended maximum: {{maxValue}}', { nsSeparator: '---', maxValue: maxValue })
```

You can confirm it works by making sure `Recommended maximum: {{maxValue}}` shows up in the `en.pot` file

Comment (Sharmyn28) [reply src/components/sections/user-sync-test/run-test.js]: Thanks, this did the trick, I checked the `en.pot `

**Pull request merged**